### PR TITLE
Add tests that non-empty administration_allowed_sids blocks non-admin GRANT

### PR DESCRIPTION
**Pull request by yurikiselev**
### Description for reviewers
Добавлены два теста, которые показывают, что непустой список `administration_allowed_sids` закрывает обход `GRANT` (`allowACLBypass = isAdmin` в `TxProxy`) даже при `enforce_user_token_requirement: false` и `enforce_user_token_check_requirement: false`.

### CODE REVIEW

File: ydb/tests/functional/security/acl/test_non_admin_grant.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+def create_user_with_minimal_grants(cluster, admin_driver_config):
+    user_name = unique_user_name("user")
+    run_with_assert(admin_driver_config, f"CREATE USER {user_name};")
+    # the least a user needs to reach the object, no 'ydb.access.grant' among them
+    provide_grants(admin_driver_config, user_name, DATABASE, ['ydb.database.connect', 'ydb.granular.describe_schema'])
+    user_driver_config = ydb.DriverConfig(
+        endpoint="%s:%s" % (cluster.nodes[1].host, cluster.nodes[1].port),
+        database=DATABASE,
+        credentials=ydb.StaticCredentials.from_user_password(user_name, ""),
+    )
```
[the remark this answers is not in the document]
Comment (yurikiselev) [reply ydb/tests/functional/security/acl/test_non_admin_grant.py:105]: Описание ревью было не точным. Мне как раз нужен кейс, когда юзер с токеном пытается поменять ACL.
Резовлю комментарий, но правкой описания ПРа, не кода теста.
